Tools/cases_generator/tier1_generator.py

```diff
@@ -41,20 +41,35 @@ def declare_variables(inst: Instruction, out: CWriter) -> None:
         if isinstance(uop, Uop):
             for var in reversed(uop.stack.inputs):
                 if var.name not in variables:
-                    type = var.type if var.type else "PyObject *"
                     variables.add(var.name)
+                    type = var.type if var.type else "PyStackRef "
```

Comment:
```suggestion
                    type = var.type if var.type else "_PyStackRef"
```
Maybe we should drop the `_` prefix, it is just clutter.
And no trailing spaces in the typenames, please.